src/components/HomePage.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import SearchBar from './SearchBar';
+
+class HomePage extends React.Component {
+  render() {
+    return (
+      <>
+        <SearchBar />
+        <p data-testid="home-initial-message">Digite algum termo de pesquisa ou escolha uma categoria.</p>
```

Comment:
Tinha que corrigir o comprimento da linha =)

```suggestion
        <p data-testid="home-initial-message">
          Digite algum termo de pesquisa ou escolha uma categoria.
        </p>
```